lua/mason-null-ls/init.lua

```diff
@@ -40,7 +40,11 @@ local function setup_handlers(handlers)
 		Optional.of_nilable(handlers[source_name]):or_(_.always(default_handler)):if_present(function(handler)
 			log.fmt_trace('Calling handler for %s', source_name)
 
+			local ignored_methods = require('mason-null-ls.settings').current.ignore_methods
 			local types = _.filter_map(function(method)
+				if vim.tbl_contains(ignored_methods, method) then
```

Comment:
Yeah, I think it just becomes more verbose to list off allowed methods. It's more typical you just want to ignore them. I'm good either way !

Definitely when building extra lazy specs such as in LazyVim `extras` and AstroCommunity it makes it easier to be additive when it's ignore because you can add those in rather than hope that the opts currently have the full `allowed_methods` populated and then to remove a method. If that makes sense.

Reply:
Here is an example of it making sense for it to be `ignore_methods` if you are interested: https://github.com/AstroNvim/astrocommunity/pull/756/files

Reply:
I opened another PR as a different approach/option for tackling this type of thing. Let me know what you think!